plugins/config/src/ConfigurationEditorWidget/components/ConfigurationEditor.js

```diff
@@ -64,13 +71,23 @@ const Member = observer(props => {
     }
     return (
       <>
-        <FormLabel>{slotName}</FormLabel>
-        <div className={classes.subSchemaContainer}>
-          {typeSelector}
-          <FormGroup>
-            <Schema schema={slot} />
-          </FormGroup>
-        </div>
+        <Accordion
+          defaultExpanded
+          className={classes.accordion}
+          TransitionProps={{ unmountOnExit: true, timeout: 0 }}
+        >
+          <AccordionSummary
+            expandIcon={<ExpandMoreIcon className={classes.expandIcon} />}
+          >
+            <Typography>Config {slotName}</Typography>
```

Comment:
I think having the `Config ` prefix on every slot ends up being a bit confusing. What I think could be nice is having the full path (e.g. "config - display 1 - renderer) on each slot.

Reply:
Could try that...Having the text "Config" as a prefix on each header was sort of to help the aesthetics and have the first letter of the section header capitalized.